src/components/supporters/Supporters/index.js

```diff
@@ -74,14 +76,18 @@ export default React.createClass({
 
   loadPages: function () {
     var endpoint = this.getEndpoint()
-    var cancelLoad = endpoint('individual', 20 + this.props.exclusions.length, this.onSuccess, { includePages: true, includeFootprint: true })
+    var cancelLoad = endpoint('individual', pagesLength + this.props.exclusions.length, this.onSuccess, { includePages: true, includeFootprint: true })
     this.setState({ cancelLoad: cancelLoad })
   },
 
   onSuccess: function (result) {
     var pages = result && result.leaderboard && result.leaderboard.pages ? result.leaderboard.pages : []
     pages = pages.filter(page => this.props.exclusions.indexOf(page.id) === -1)
```

Comment:
@ellsclytn What are your thoughts about chucking `slice(0, 20)` at the end here (or where the state is set) rather than an if statement?